packages/amplify-cli-core/API.md

```diff
@@ -781,6 +785,9 @@ export type ErrorParameter = {
 // @public (undocumented)
 export type EventPrefix = 'pre' | 'post';
 
+// @public (undocumented)
+export const execAsStringPromise: (command: string, opts?: execa.Options) => Promise<string>;
```

Comment:
nit: I'd call it `execWithOutputAsString` or `execWithStringOutput`.

Reply:
done